README.md

```diff
@@ -147,95 +152,137 @@ MODEL_NAME = "user123 customer churn model"
+Export the "accessToken" part of this response in the terminal window as `WML_AUTH_TOKEN`. Get the `URL` from the *API reference* by copying the `Endpoint`, and export it as `URL`:
 
-![Testing the deployed model](doc/source/images/testing-1-api.png)
+![Model Deployment Endpoint](doc/source/images/ModelDeploymentEndpoint.png)
 
-### Test the deployed model with cURL
+```bash
+export WML_AUTH_TOKEN=<value-of-access-token>
+export URL=https://blahblahblah.com
+```
 
-Clicking the *Generate Code* button will pop open a window with some code for you to copy. The code will use the cURL command line utility to test the REST APIs. Here's an example of the generated code that can be run from a terminal window with the `curl` command.
+Now run this curl command from a terminal window:
 
 ```bash
-curl -k -X POST \
-  https://169.48.4.137:31843/dmodel/v1/telco-churn/pyscript/telco-model/score \
-  -H 'Authorization: Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VybmFtZSI6InNjb3R0ZGEiLCJwYWNrYWdlTmFtZSI6InRlbGNvLWNodXJuLTEiLCJwYWNrYWdlUm91dGUiOiJ0ZWxjby1jaHVybiIsImlhdCI6MTU2OTk2NTcyNn0.Xsj-wksYFVvxOpnamO97WESMgrbeh4mSh4qeIjzoc5_VYQahPoTwR14nM5vTqCpUI9kg9HRS7SIavFkeH9S-aR4z1FWibFj_Sbcnaib2uSbHZwX3OZvCxfriUxZn_mAYtQFR350LgjJwctI1I4tl4yi7G4J92SDvv3h2sVAy-5n9-0xUmmSmze8lPdCmejTAsus7wntQqCF6a0ON2RmSTX1VV1H0BBH6oxioYjKLRdRU5tXXyrYUacveTvc2B2eySAYkLV0xU5FNIRvWdwGxYtpXwb8jr_MCKTuCGBicqmu5JemQFucvSU7yGDmlq1OmTOAhm2MKFi6lv1WRLIGNlA' \
-  -H 'Cache-Control: no-cache' \
-  -H 'Content-Type: application/json' \
-  -d '{"args":{"input_json":[{"gender":"Female","SeniorCitizen":0,"Partner":"No","Dependents":"No","tenure":1,"PhoneService":"No","MultipleLines":"No phone service","InternetService":"DSL","OnlineSecurity":"No","OnlineBackup":"No","DeviceProtection":"No","TechSupport":"No","StreamingTV":"No","StreamingMovies":"No","Contract":"Month-to-month","PaperlessBilling":"No","PaymentMethod":"Bank transfer (automatic)","MonthlyCharges":25.25,"TotalCharges":25.25}]}}'
+curl -k -X POST --header 'Content-Type: application/json' --header 'Accept: application/json' --header "Authorization: Bearer  $WML_AUTH_TOKEN" -d '{"input_data": [{"fields": ["gender","SeniorCitizen","Partner","Dependents","tenure","PhoneService","MultipleLines","InternetService","OnlineSecurity","OnlineBackup","DeviceProtection","TechSupport","StreamingTV","StreamingMovies","Contract","PaperlessBilling","PaymentMethod","MonthlyCharges","TotalCharges"],"values": [["Female",0,"No","No",1,"No","No phone service","DSL","No","No","No","No","No","No","Month-to-month","No","Bank transfer (automatic)",25.25,25.25]]}]}' $URL
 ```
 
+A json string will be returned with the response, including a "Yes" of "No" at the end indicating the prediction of if the customer will churn or not.
+
 ## 9. Create a Python Flask app that uses the model
```

Comment:
shouldn't say "workshop" in a code pattern

Reply:
done.